flink-table/flink-sql-parser/pom.xml

```diff
@@ -51,15 +51,26 @@ under the License.
 			<version>${calcite.version}</version>
 			<exclusions>
 				<!--
+				"mvn dependency:tree" as of Calcite 1.20:
 
-				Dependencies that are not needed for how we use Calcite right now.
-
-				"mvn dependency:tree" as of Calcite 1.18.0:
-
-				[INFO] +- org.apache.calcite:calcite-core:jar:1.18.0:compile
-				[INFO] |  +- org.apache.calcite.avatica:avatica-core:jar:1.13.0:compile
-				[INFO] |  \- com.google.guava:guava:jar:19.0:compile
+				[INFO] +- org.apache.calcite:calcite-core:jar:1.20.0:compile
+				[INFO] |  +- org.apache.calcite.avatica:avatica-core:jar:1.15.0:compile
+				[INFO] |  +- org.apache.calcite:calcite-linq4j:jar:1.20.0:compile
+				[INFO] |  +- org.apache.commons:commons-dbcp2:jar:2.6.0:compile
+				[INFO] |  +- org.apache.commons:commons-lang3:jar:3.8:compile
+				[INFO] |  +- com.esri.geometry:esri-geometry-api:jar:2.2.0:compile
+				[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.9.8:compile
+				[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.9.8:compile
+				[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.9.8:compile
+				[INFO] |  +- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:jar:2.9.8:compile
+				[INFO] |  +- com.google.guava:guava:jar:19.0:compile
+				[INFO] |  +- com.jayway.jsonpath:json-path:jar:2.4.0:compile
```

Comment:
We can exclude all these useless packages. Only `avatica-core` and `guava` are enough.